UnicornRender/video/source/PerspectiveCamera.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/PerspectiveCamera.hpp>
+
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+
+PerspectiveCamera::PerspectiveCamera(system::Window* window, glm::mat4& cameraProj) :
+    CameraProjection(window, cameraProj),
+    m_fov(45.f),
+    m_fovLowerBound(44.f),
+    m_fovUpperBound(45.f),
+    m_znear(0.1f),
+    m_zfar(1000.f)
```

Comment:
put all commas before each member so that when you add more variables, diff will be shorter